modules/icress_module.php

```diff
@@ -39,26 +39,28 @@ function icress_getJadual() {
 }
 
 function icress_getCampus($campus, $faculty) {
-		$form_names = getFormNames();
+		// $form_names = getFormNames();
 
 		$postdata = http_build_query(
 				array(
-						$form_names['search_campus'] => $campus,
-						$form_names['search_faculty'] => $faculty
+						// $form_names['search_campus'] => $campus,
+						// $form_names['search_faculty'] => $faculty
+						'search_campus' => $campus,
+						'search_course' => $faculty,
 				)
 		);
 		
 		$options = array('http' =>
 				array(
 						'method'  => 'POST',
-						'header'  => 'Content-Type: application/x-www-form-urlencoded',
+						'header'  => "Content-Type: application/x-www-form-urlencoded\nReferer: https://simsweb4.uitm.edu.my/estudent/class_timetable/index.htm",
```

Comment:
Without `referer` header, They won't response your HTTP request. You will get empty page